scripts/show_settings.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python3
+
+import sys
+import shub.settings as cfg
+
+if __name__ == "__main__":
```

Comment:
This is closer! You'd actually want it to be a proper Django management module - this would allow the user to run like `python manage.py show_settings`. Here is more background https://docs.djangoproject.com/en/4.2/howto/custom-management-commands/ and there is an example [here](https://github.com/singularityhub/sregistry/tree/master/shub/apps/base/management/commands). Basically, you add a script of similar format to that directory, then test running with `manage.py` and add a note to the documentation about running it. So if you had a script called `show_settings.py` in that directory, and it followed the same format as that registry.py example, you could run `python manage.py show_settings.py` let me know if that makes sense!

Reply:
i'll get django on my hands...  i'll give it a try...

Reply:
Awesome! And if you run into trouble since it's about to be the weekend I can give it a shot too, and at that time feel free to assign me other tasks for stuffs that you are having trouble with. Django is a bunch to learn if you haven't used it before so I'm totally happy to help!

Reply:
think i got this going.  had to pull in the whole dependency list for django it seemed...